config_defaults.yml

```diff
@@ -503,6 +503,11 @@ Gemfile:
         condition: "Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.1.0')"
       - gem: fast_gettext
         condition: "Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.1.0')"
+      # add the changelog gem to puppet-module-gems, when there is a release
+      - gem: 'github_changelog_generator'
+        git: 'https://github.com/skywinder/github-changelog-generator'
+        ref: 'master'
```

Comment:
* This introduces a dependency that's not part of the PDK package, which will break offline usage
* Instead of a branch name, this should list a SHA to avoid any update to GCG break our users.